tools/ci_build/github/azure-pipelines/templates/android-java-api-aar-test.yml

```diff
@@ -59,7 +64,17 @@ jobs:
     workingDirectory: $(Build.BinariesDirectory)
 
   # skip emulator tests for qnn package as there are no arm64-v8a emulators and no qnn libraries for x86
-  - ${{ if not(contains(parameters.packageName, 'qnn')) }}:
+  # skip emulator tests for webgpu package as software rendering with swiftshader does not work without host GPU
+  - script: |
+        set -e -x
```

Comment:
the output with `set -x` can cause issues for setting variables with VSO commands.

see this note:
https://github.com/microsoft/onnxruntime/blob/1cb53d0054774db37ed500e43988497c66599c11/tools/ci_build/github/azure-pipelines/templates/use-android-ndk.yml#L20